### ISSUE TITLE
Restructure repo for usability

### ISSUE THREAD

**Issue by milroy** [flux-framework/flux-k8s#37]
The current `main` branch contains out of date instructions and examples. We need to update it to make it usable. Items to consider:

- replace the contents of the `examples` directory with examples from the CANOPIE22 paper
- update the README to include build and running instructions from the CANOPIE22 paper

Other suggestions are welcome.

**Comment by vsoch** [comment]
Whatever you and @cmisale decide, I can give it a test run and give feedback if anything is unclear since I haven’t used anything here. 

I’m not sure about the content of the paper, but if there is a container build or similar it might make sense to automate that or at least make sure there is one available alongside the repository.

TLDR let me know how I can help.

**Comment by vsoch** [comment]
I think we are good to close here - we have good  documentation in the README for deployment (from our containers) and custom builds, along with instructions for a pod example, and TBA job example (used for testing) that will come with https://github.com/flux-framework/flux-k8s/pull/49
